### ISSUE TITLE
Redesign: does 'Training + Workshops' work for screen readers?

### ISSUE THREAD

**Issue by Cleop** [dwyl/dwyl-site#243]
@iteles @jsms90 @nelsonic, accessibility gurus... do you know how a screen reader pronounces a '+', I presume it's as a 'plus'? For this reason is it discouraged to use plus to mean 'and'?

E.g.
<img src="https://user-images.githubusercontent.com/16775804/26936391-7f3cb296-4c66-11e7-826d-2763af770c6a.png" width=250px />

**Comment by jsms90** [comment]
@Cleop Depends on the brand of screen reader

https://www.webassign.net/manual/student_guide/c_a_screen_reader_config.htm
> Often, screen readers are configured not to announce minus signs or hyphens. Notation like -x or −x should be announced as minus x.

https://www.deque.com/blog/dont-screen-readers-read-whats-screen-part-1-punctuation-typographic-symbols/
> + The NVDA screen reader, for instance, doesn't read hardly any typographical symbols at all in its default configuration, making common symbols like asterisks and plus symbols essentially useless to NVDA users. This means web developers can't use an asterisk to denote a required field on a form, unless they supplement the asterisk with some other NVDA-friendly method.
> + Similarly, NVDA ignores the plus and equals symbols. It says "five two seven" when it should say "five plus two equals seven," making it hard -- or impossible -- to write basic math expressions.
JAWS reads the plus symbol correctly, but not the minus symbol. Jaws incorrectly says "five dash two" when it should say "five minus two," even when using the HTML entity &minus; (not a regular dash) to specify the minus symbol.
> + JAWS also reads the "less than or equal to" symbol incorrectly. JAWS says "equals," which is wrong. Between NVDA and JAWS, web authors who want to write even the most basic of arithmetic expressions -- one plus one, or one minus one, for example -- can't count on screen readers to read them right. That's a serious limitation.
> + On the Mac side, VoiceOver doesn't inform users about quotation marks, parentheses, or dashes, making it so users don't know when quotation or parenthetical phrases begin or end. To its credit, VoiceOver does pause briefly for these elements, but users won't know the difference between a dash, parentheses, or a quotation, because all pauses sound the same, even though the reasons for the pauses are all different, and should convey different semantic meanings.

**Comment by jsms90** [comment]
and browser apparently

[JAWS tests](http://lab.dotjay.co.uk/tests/screen-readers/character-entities/#results-4)

**Comment by Cleop** [comment]
Thanks @jsms90 🙌 , looks like none of them say anything resembling 'and', so in which case it wouldn't seem good practice to use the plus in substitution for an 'and' or '&'. I'll share this with the team 👍

**Comment by Cleop** [comment]
@iteles is there a way we can test this? or should it be closed without testing?

**Comment by iteles** [comment]
We can totally test this. Just grab a free screen reader and test it, record your test using Quicktime and drop it into this issue before closing please 😊

**Comment by markwilliamfirth** [comment]
I'll test so Cleo can have more dev time, but if you really want to test it Cleo, then happy for you to!

**Comment by Cleop** [comment]
@markwilliamfirth please go ahead 👍

**Comment by ZooeyMiller** [comment]
I just tested this using [ChromeVox](https://chrome.google.com/webstore/detail/chromevox/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en) and "+" is definitely read out as "plus". I tried to do a screen recording but I don't have decent screen recording software with audio, so maybe someone with a mac could do the screen recording? 

"&" is read out as "and" by chromevox so maybe that's a reasonable replacement.

**Comment by markwilliamfirth** [comment]
@ZooeyMiller ok happy for you to replace with &amp; if that works!

**Comment by markwilliamfirth** [comment]
Just tested actual site with ChromeVox and it says "Training and plus workshops"

**Comment by markwilliamfirth** [comment]
Have replaced with & as solution
